src/TRE-API/Controllers/PostgrsCredentialsController.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace TRE_API.Controllers
+{
+    /// <summary>
+    /// API controller for managing postgres credentials and authentication operations
+    /// </summary>
+    [Route("api/v1/postgrescredentials")]
+    [ApiController]
+    [SwaggerTag("Postgrescredentials", "Manage Postgres Database Credentials")]
+    [Produces("application/json")]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+    public class PostgrsCredentialsController : ControllerBase
```

Comment:
```suggestion
    public class PostgresCredentialsController : ControllerBase
```

Typo (in the filename too)

Reply:
the class name is fixed but the filename is still incorrect